twa-matching.md

```diff
@@ -0,0 +1,24 @@
+# Project TWA Careers and Candidates Matching
+
+## Client Profile
+
+The Transformative Workforce Academy (TWA) is an initiative at Saint Louis University aimed at supporting individuals at risk of incarceration to lead productive lives without returning to prison. SLU's comprehensive approach includes prevention, in-prison, and reentry services. TWA specifically focuses on helping individuals leaving prison to find and maintain meaningful employment.
+
+TWA started its efforts in 2018 by organizing the Second Chance Job Fair in partnership with federal and state probation and parole offices. The fair aimed to connect job seekers with employers who consider candidates holistically, beyond their criminal records. Building on the success of these job fairs, TWA now aims to provide year-round support to jobseekers and employers through the Second Chance Job Portal. The Second Chance Job Portal will offer resources and assistance for jobseekers seeking second-chance employment, as well as support for employers interested in hiring and retaining this often overlooked segment of the workforce. The goal is to facilitate successful employment opportunities and help address the root causes of crime by providing ongoing support and resources to those seeking a second chance.
+
+## Project Brief
+
+This project involves collecting and matching the below information
+
+1. criminal records of individuals
+2. Job requirements from the hiring team.
+
+If the above 1 and 2 requirements are not related to each other. then we can send this particular profile to hiring team. similarly vice versa.
```

Comment:
In Project Brief you can add the following points :
Client Profile
Project Goals
Second Chance Job Portal Overview
Issue: Ensuring Effective Matching between Candidates and Job Requirements

Functional Requirements can be more specific as follows:
1.)Profile Building
2.)Job Posting
3.)Candidate Application and Job Search
4.)Intelligent Matching Algorithm
5.)Filtering and Sorting Options
6.)Candidate Training and Development
7.)Feedback and Reporting